src/store/edge-device.js

```diff
@@ -0,0 +1,29 @@
+import { FETCH_EDGE_DEVICE_DETAILS } from './action-types.js'
+import { EDGE_DEVICE_DETAILS } from './mutation-types'
+
+const state = {
+  version: require('@/../package.json').version
+}
+
+const mutations = {
+  [EDGE_DEVICE_DETAILS] (state, edgeDetails) {
+    state.version = edgeDetails.version
+  }
+}
+
+const actions = {
+  [FETCH_EDGE_DEVICE_DETAILS] ({ commit }, details) {
```

Comment:
@vickywane this action is not very useful, because it only commits a single mutation. 
Your intention was probably to use it as a mechanism to call the edgeAPI and subscribe to mutation notifications in the vue code. Please implement that or simply remove the action and only use the mutation in the vue  code.

Reply:
I have removed this action and called the commit method directly from the concerned component and test suite.